aries_cloudagent/protocols/present_proof/v1_0/routes.py

```diff
@@ -484,11 +483,12 @@ async def presentation_exchange_list(request: web.BaseRequest):
     }
 
     try:
-        records = await V10PresentationExchange.query(
-            session=session,
-            tag_filter=tag_filter,
-            post_filter_positive=post_filter,
-        )
+        async with context.session() as session:
+            records = await V10PresentationExchange.query(
+                session=session,
```

Comment:
where does this session come from, since the delta doesn't create it on line 475?